positions/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+
+import datetime
+
+# Create your models here.
+
+class Position(models.Model):
+    """
+    Describes a leadership position for a specific time. A new position instance
+    should be created every time one needs to be filled.
+    """
+    name = models.CharField(verbose_name="Position Name", max_length=64,
+            null=False, blank=False)
+    description = models.TextField(verbose_name="Position Description",
+            null=False, blank=False)
+    position_start = models.DateField(verbose_name="Term Start", null=False,
+            blank=False)
+    position_end = models.DateField(verbose_name="Term End", null=False,
+            blank=False)
+    reports_to = models.ForeignKey(get_user_model(), on_delete=models.CASCADE,
+            blank=True, null=True)
+    closes = models.DateField(verbose_name="Applications Close", null=True,
+            blank=True)
+    application_form = models.CharField(verbose_name="Link to external application form", null=False, blank=False, max_length=128)
```

Comment:
I didn't know about URLField! Thank you

Reply:
Also, I made it non-nullable because this app currently doesn't track applications. It requires the use of an external tool to do that.

Reply:
Also, I made it non-nullable because this app currently doesn't track applications. It requires the use of an external tool to do that.

Reply:
Right, but not all positions should require an application. When we have exec board elections in C term, no one applies for that.

Reply:
Are you anticipating making a DB position for exec board positions? I think it'd be better to just send them to the bylaws

Reply:
So in the past, we have actually used the current version of this feature to provide details about exec positions leading up to elections and from what I've heard, it seemed to be pretty helpful. The problem with just sending people to the bylaws is that the bylaws don't really provide a lot of detail on most of the positions and up until recently were quite outdated. Regardless of whether we choose to use this for exec board elections this year or not, I've found that it's usually best to support capabilities like this anyways. It's generally a good idea to assume that a future exec board might choose to use this feature differently and if we plan for that now, it will make everything much easier.

Reply:
Resolved in b926a9f